nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/PutMongo.java

```diff
@@ -288,17 +288,28 @@ protected WriteConcern getWriteConcern(final ProcessContext context) {
             writeConcern = WriteConcern.UNACKNOWLEDGED;
             break;
         case WRITE_CONCERN_FSYNCED:
-            writeConcern = WriteConcern.FSYNCED;
+            writeConcern = WriteConcern.JOURNALED;
+            getLogger().warn("Using deprecated write concern FSYNCED");
```

Comment:
`getWriteConcern()` gets called in `onTrigger()` so I think this is going to be really chatty. It might be helpful to just do this warning in the `@onScheduled` in the abstract class. It's not using EL so it doesn't even need to be called in `onTrigger()`.